package.json

```diff
@@ -20,17 +20,15 @@
     "stylelint-csstree-validator": "^1.9.0",
     "stylelint-scss": "^3.21.0",
     "webpack": "^5.74.0",
-    "webpack-cli": "^4.10.0",
-    "webpack-dev-server": "^4.9.0"
+    "webpack-cli": "^4.10.0"
   },
   "version": "1.0.0",
-  "description": "Simple To Do List Application",
-  "name": "To Do List",
+  "description": "My webpack project",
+  "name": "my-webpack-project",
   "scripts": {
```

Comment:
Please add a script that can be used to serve your application locally, you can do this by installing `webpack-dev-server` using `npm` and adding this line in the scripts section of `package.json`
```JavaScript
  "scripts": {
    "start": "webpack serve --open"
  }
```

Once this is done you can run your application by running `npm run start`